packages/Organisms/SiteHeader/search.vue

```diff
@@ -76,6 +77,12 @@ export default {
       margin: $rpl-site-header-search-margin-xl;
     }
 
+    &__form {
+      &.rpl-search-form {
+        padding: 0;
```

Comment:
This is to clear the default search form padding. It's not needed when the form is used in the header-search component.

Reply:
Hi @alan-cole Just confirm with you, that change won't break any existing search forms like those on search result and event search page, right?

Reply:
This should not affect those cases as the SCSS would compile down to:

```
.rpl-site-header-search__form.rpl-search-form {
  padding: 0;
}
```

which is adding a header-specific class. I have checked on the search results page (you can see it in the screenshot) and it has kept the search form component padding as needed.